charts/backingservices/values.yaml

```diff
@@ -44,14 +44,14 @@ srs:
     #port: 9200
     #protocol: https
     # The elasticsearch connection supports three authentication methods: basic authentication , AWS IAM role-based authentication and Elasticsearch secure connection(tls).
-    # To configure tls on internal cluster uncomment and set `srs.srsStorage.tls.enabled: true`
+    # set `srs.srsStorage.tls.enabled: true` to enable tls on internal/external elasticsearch cluster.
     tls:
      enabled: false
     # Specify the certificate name used to connect to elasticsearch when tls.enabled: true and srsStorage.provisionInternalESCluster: false.
-    # certificateName: "elastic-certificates.p12"
-    # To configure basicAuthentication on internal cluster uncomment and set `srs.srsStorage.basicAuthentication.enabled: true`
+    # certificateName: "Certificate_Name"
+    # set `srs.srsStorage.basicAuthentication.enabled: true` to enable basicAuthentication on internal/external elasticsearch.
     basicAuthentication:
-      enabled: false
+      enabled: true
     #     when  basic authentication/tls based authentication is configured to your externally-managed Elasticsearch cluster, uncomment and add the
```

Comment:
To configure basic authentication or TLS-based authentication to your externally-managed Elasticsearch service in your SRS cluster, uncomment and add the parameter details: srs.srsStorage.esCredentials.username and srs.srsStorage.esCredentials.password.